automation_tools/__init__.py

```diff
@@ -384,6 +370,48 @@ def setup_default_capsule(interface=None, run_katello_installer=True):
         return installer_options
 
 
+def setup_default_libvirt(bridge=None):
+    """Task to setup a the default capsule for Satellite
+
+    :param str interface: Network interface name to be used
+    """
+    run('yum install -y libvirt libvirt-daemon-kvm virt-install qemu-kvm')
+    run('lsmod | grep kvm_')
+    run('sed -i \'s/^#*\s*LIBVIRTD_ARGS=.*/LIBVIRTD_ARGS=--listen/\''
+        ' /etc/sysconfig/libvirtd')
+    run('sed -i \'s/^#*\s*listen_tls\s*=.*/listen_tls = 0/\''
+        ' /etc/libvirt/libvirtd.conf')
+    run('sed -i \'s/^#*\s*listen_tcp\s*=.*/listen_tcp = 1/\''
+        ' /etc/libvirt/libvirtd.conf')
+    run('sed -i \'s/^#*\s*auth_tcp\s*=.*/auth_tcp = "none"/\''
+        ' /etc/libvirt/libvirtd.conf')
+    # manage_daemon('enable', 'libvirtd')
```

Comment:
libvirtd is by default enabled. After running ```yum install libvirt``` systemctl says libvirt is enabled.

I may uncomment it to enable libvirt for sure :)